docs/contributing/git-style.rst

```diff
@@ -49,8 +49,21 @@ features or plugins to streamline this process (e.g.,
 It is also possible to ask GitHub to squash your commits for you. This is the
 "squash and merge" option when merging your PR.
 
-Commit Message Style
-====================
+Pull Request Title Style
+========================
+
+Include the Jira issue key(s) that are being addressed at the end of
+the pull request title as a comma separated list in brackets. Example PR title::
+
+  Reduce memory usage of Foo [KAT-1000, KAT-1001]
+
+Commit/Pull Request Message Style
```

Comment:
I'd probably just write

```
Pull Request Style
==============

Include the Jira issue key(s) that are being addressed at the end of
the pull request title as a comma separated list in brackets.

Example PR title::

  Reduce memory usage of Foo [KAT-1000, KAT-1001]

If you squash and merge your PR, by default, the PR description and
title are turned into the commit message for your squashed PR, so apply
the recommendations about commit message style below to your PR description instead.

Commit Message Style
==================

If you intend to rebase and merge, please still include Jira issue keys as described in
pull request style above.

If you intend to squash and merge, apply the recommendations in this section to your
PR message instead of your commit message.
```

Reply:
Thanks for the better language